src/unity/toolkits/drawing_classifier/mps_drawing_classifier.mm

```diff
@@ -0,0 +1,16 @@
+#import <unity/toolkits/drawing_classifier/mps_drawing_classifier.hpp>
+
+@implementation DrawingClassifierModel
+
+- (id _Nullable)initWithParameters:(NSString *_Nonnull)name
+                         inputNode:(MPSNNImageNode *_Nonnull)inputNode
+                            device:(id<MTLDevice> _Nonnull)dev
+                         cmd_queue:(id<MTLCommandQueue> _Nonnull)cmd_q
+                       initWeights:(std::map<std::string, std::vector<float>>)weights {
+  @autoreleasepool {
+    self = [super init];
+    return self;
+  }
+}
```

Comment:
Also seems like this is missing a lot. I would expect to see the actual network definition here, but I'm not seeing that any where.

Reply:
I forgot to push a commit.

Reply:
I pushed it's under the commit history but it's not showing up here. Hmm.